.changeset/four-parents-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/frontend-app-api': minor
```

Comment:
Should we callout in this changeset that `createApp` and `CreateAppFeatureLoader` is now deprecated and should be imported from `@backstage/frontend-defaults` instead?

Reply:
Yep! 👍 done in https://github.com/backstage/backstage/pull/26290/commits/47b4e578c0d4aec8336e886cd237c179eb3f03dd